### ISSUE TITLE
fix: don't permit account deletion if user owns servers

### ISSUE THREAD

**Issue by insertish** [stoatchat/stoatchat#315]

**Comment by erkinalp** [comment]
alternative: make guilds from deleted users unowned

**Comment by maxgorovenko** [comment]
I'm not sure if I understand correct, but do you mean the operation of removing a member from the server? I didn't find the operation of removing the user.

**Comment by erkinalp** [comment]
Yep, removing the owner leaving the guild ownerless

**Comment by maxgorovenko** [comment]
Did PR, hope this is what we need :) https://github.com/revoltchat/backend/pull/380

**Comment by ThomasJRyan** [comment]
This is implemented

**Comment by insertish** [comment]
I don't think the backend part of this is implemented yet?
There is just a client side block for the currently.
This can only be solved once authifier is merged into this code base.

**Comment by IAmTomahawkx** [comment]
blocked on #658
